pom.xml

```diff
@@ -1498,6 +1506,22 @@
                     </exclusion>
                 </exclusions>
             </dependency>
+
+            <dependency>
+                <groupId>com.qcloud.cos</groupId>
+                <artifactId>hadoop-cos</artifactId>
+                <version>${cos.hadoop.version}</version>
+            </dependency>
+            <dependency>
+                <groupId>com.qcloud</groupId>
+                <artifactId>cos_api-bundle</artifactId>
+                <version>${cos.bundle.version}</version>
+            </dependency>
+            <dependency>
+                <groupId>com.qcloud</groupId>
+                <artifactId>dlc-data-catalog-metastore-client</artifactId>
+                <version>${dlc.client.version}</version>
+            </dependency>
         </dependencies>
```

Comment:
Whether licenses are required for such dependencies？

Reply:
I will add it.

Reply:
dlc does not have license